docs/reference/usage.qmd

```diff
@@ -60,7 +60,8 @@ While physically incorrect, it is useful for a first correctness check on a mode
 ## Allocation settings
 Currently there are the following allocation settings:
 - `use_allocation`: A boolean which says whether allocation should be used or not;
-- `timestep`: a float value in seconds which dictates the update interval for allocations.
+- `timestep`: A float value in seconds which dictates the update interval for allocations;
+- `default_source_priority`: An integer per source type for the allocation algorithm: `user_demand`, `boundary`, `level_demand`, `flow_demand`, `subnetwork_inlet`. Flow boundaries and level boundaries are combined in the single category `boundary`. By default all nodes of the same type have the same source priority, so that all source nodes of the same node type within a subnetwork are optimized for at once.
```

Comment:
Why do we share the flow and level boundaries? Can't we distinguish between them?